### DISCUSSION TITLE
Page title is not updated

### DISCUSSION THREAD

**Discussion by ghost** [community/community#63132] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

**To reproduce:**

1. Open the user profile whose timeline contains a box (a card about a created PR/issue)
2. Open this PR/issue
3. Click the Code tab
4. The page title is not updated

**Comment by ViridianTelamon** [comment]
You could try to delete the file and create a new one with the new title, or try to remove it from your profile and then add it back after, it should should then.

**Reply by ghost** [reply]
What file you are talking about?

**Reply by ViridianTelamon** [reply]
The file that is directly connected to your profile statistics.
